doc-content/enterprise-only/installation/ssh-configuring-proc.adoc

```diff
@@ -0,0 +1,86 @@
+[id='ssh-configuring-proc_{context}']
+
+= Configuring SSH to use RSA
+SSH is used to clone Git repositories. By default, the DSA encryption algorithm is provided by Business Central. However, some SSH clients, for example SSH clients in the Fedora 23 environment, use the RSA algorithm instead of the DSA algorithm. Business Central contains a system property that you can use to switch from DSA to RSA if required. 
+
+[NOTE]
+====
+SSH clients on supported systems, for example Red Hat Enterprise Linux 7, are not affected by this issue.
```

Comment:
Could we add a link to all supported systems or is it an overkill?

Reply:
I'm not aware of a link to 6.4 supported systems, just 7.x.

Reply:
But did added the link for the 7.x books.